"A negative value is not valid" for height and width

**Describe the bug**
We seem to send this to the console, but

**Logs**
<details>
```
react-dom.production.min.js:28 Error: <svg> attribute width: A negative value is not valid. ("-24")
Z @ react-dom.production.min.js:28
ta @ react-dom.production.min.js:201
Mc @ react-dom.production.min.js:247
mc @ react-dom.production.min.js:246
bc @ react-dom.production.min.js:246
cc @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
lc @ react-dom.production.min.js:240
notify @ Subscription.js:16
notifyNestedSubs @ Subscription.js:88
i @ Subscription.js:93
g @ redux.js:296
(anonymous) @ session-middleware.js:53
(anonymous) @ middleware.js:22
(anonymous) @ index.js:28
(anonymous) @ store.js:21
dispatch @ redux.js:691
(anonymous) @ redux.js:206
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
Promise.then (async)
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ redux.js:202
(anonymous) @ store.js:19
dispatch @ redux.js:691
(anonymous) @ data-fetching.js:382
(anonymous) @ data-fetching.js:380
(anonymous) @ redux.js:204
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ redux.js:202
(anonymous) @ store.js:19
n.<computed> @ bindActionCreators.js:8
(anonymous) @ Dashboard.jsx:142
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
_R @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
ua @ react-dom.production.min.js:212
yc @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
zc @ react-dom.production.min.js:248
cc @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
lc @ react-dom.production.min.js:240
notify @ Subscription.js:16
notifyNestedSubs @ Subscription.js:88
i @ Subscription.js:93
g @ redux.js:296
(anonymous) @ session-middleware.js:53
(anonymous) @ middleware.js:22
(anonymous) @ index.js:28
(anonymous) @ store.js:21
dispatch @ redux.js:691
(anonymous) @ redux.js:206
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
Promise.then (async)
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ redux.js:202
(anonymous) @ store.js:19
n.<computed> @ bindActionCreators.js:8
(anonymous) @ Dashboard.jsx:170
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
_R @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ Dashboard.jsx:123
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
_R @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
ua @ react-dom.production.min.js:212
yc @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
zc @ react-dom.production.min.js:248
cc @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rc @ react-dom.production.min.js:230
enqueueSetState @ react-dom.production.min.js:132
v.setState @ react.production.min.js:12
(anonymous) @ Router.js:109
(anonymous) @ createTransitionManager.js:228
(anonymous) @ createTransitionManager.js:83
(anonymous) @ AsyncUtils.js:74
(anonymous) @ AsyncUtils.js:80
(anonymous) @ getComponents.js:6
(anonymous) @ getComponents.js:30
(anonymous) @ AsyncUtils.js:79
m @ AsyncUtils.js:78
x @ getComponents.js:29
u @ createTransitionManager.js:77
l @ AsyncUtils.js:47
(anonymous) @ TransitionUtils.js:88
(anonymous) @ TransitionUtils.js:110
(anonymous) @ routes.jsx:179
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
Rkt @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
Promise.then (async)
Rkt @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ routes.jsx:176
o @ TransitionUtils.js:43
(anonymous) @ TransitionUtils.js:114
(anonymous) @ TransitionUtils.js:84
l @ AsyncUtils.js:34
g @ AsyncUtils.js:51
r @ TransitionUtils.js:83
runEnterHooks @ TransitionUtils.js:107
(anonymous) @ createTransitionManager.js:70
r @ TransitionUtils.js:74
runChangeHooks @ TransitionUtils.js:131
l @ createTransitionManager.js:67
(anonymous) @ createTransitionManager.js:45
l @ AsyncUtils.js:41
g @ AsyncUtils.js:51
F @ matchRoutes.js:231
c @ createTransitionManager.js:41
r @ createTransitionManager.js:222
listen @ createTransitionManager.js:246
componentWillMount @ Router.js:102
_o @ react-dom.production.min.js:137
$s @ react-dom.production.min.js:176
Oa @ react-dom.production.min.js:263
mc @ react-dom.production.min.js:246
bc @ react-dom.production.min.js:246
cc @ react-dom.production.min.js:239
rc @ react-dom.production.min.js:230
jc @ react-dom.production.min.js:281
(anonymous) @ react-dom.production.min.js:284
uc @ react-dom.production.min.js:240
Gc @ react-dom.production.min.js:284
t.render @ react-dom.production.min.js:290
T3 @ app.js:78
(anonymous) @ app.js:111
react-dom.production.min.js:28 Error: <svg> attribute height: A negative value is not valid. ("-24")
Z @ react-dom.production.min.js:28
ta @ react-dom.production.min.js:201
Mc @ react-dom.production.min.js:247
mc @ react-dom.production.min.js:246
bc @ react-dom.production.min.js:246
cc @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
lc @ react-dom.production.min.js:240
notify @ Subscription.js:16
notifyNestedSubs @ Subscription.js:88
i @ Subscription.js:93
g @ redux.js:296
(anonymous) @ session-middleware.js:53
(anonymous) @ middleware.js:22
(anonymous) @ index.js:28
(anonymous) @ store.js:21
dispatch @ redux.js:691
(anonymous) @ redux.js:206
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
Promise.then (async)
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ redux.js:202
(anonymous) @ store.js:19
dispatch @ redux.js:691
(anonymous) @ data-fetching.js:382
(anonymous) @ data-fetching.js:380
(anonymous) @ redux.js:204
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ redux.js:202
(anonymous) @ store.js:19
n.<computed> @ bindActionCreators.js:8
(anonymous) @ Dashboard.jsx:142
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
_R @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
ua @ react-dom.production.min.js:212
yc @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
zc @ react-dom.production.min.js:248
cc @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
lc @ react-dom.production.min.js:240
notify @ Subscription.js:16
notifyNestedSubs @ Subscription.js:88
i @ Subscription.js:93
g @ redux.js:296
(anonymous) @ session-middleware.js:53
(anonymous) @ middleware.js:22
(anonymous) @ index.js:28
(anonymous) @ store.js:21
dispatch @ redux.js:691
(anonymous) @ redux.js:206
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
Promise.then (async)
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ redux.js:202
(anonymous) @ store.js:19
n.<computed> @ bindActionCreators.js:8
(anonymous) @ Dashboard.jsx:170
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
_R @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ Dashboard.jsx:123
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
_R @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
ua @ react-dom.production.min.js:212
yc @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
zc @ react-dom.production.min.js:248
cc @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rc @ react-dom.production.min.js:230
enqueueSetState @ react-dom.production.min.js:132
v.setState @ react.production.min.js:12
(anonymous) @ Router.js:109
(anonymous) @ createTransitionManager.js:228
(anonymous) @ createTransitionManager.js:83
(anonymous) @ AsyncUtils.js:74
(anonymous) @ AsyncUtils.js:80
(anonymous) @ getComponents.js:6
(anonymous) @ getComponents.js:30
(anonymous) @ AsyncUtils.js:79
m @ AsyncUtils.js:78
x @ getComponents.js:29
u @ createTransitionManager.js:77
l @ AsyncUtils.js:47
(anonymous) @ TransitionUtils.js:88
(anonymous) @ TransitionUtils.js:110
(anonymous) @ routes.jsx:179
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
Rkt @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
Promise.then (async)
Rkt @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ routes.jsx:176
o @ TransitionUtils.js:43
(anonymous) @ TransitionUtils.js:114
(anonymous) @ TransitionUtils.js:84
l @ AsyncUtils.js:34
g @ AsyncUtils.js:51
r @ TransitionUtils.js:83
runEnterHooks @ TransitionUtils.js:107
(anonymous) @ createTransitionManager.js:70
r @ TransitionUtils.js:74
runChangeHooks @ TransitionUtils.js:131
l @ createTransitionManager.js:67
(anonymous) @ createTransitionManager.js:45
l @ AsyncUtils.js:41
g @ AsyncUtils.js:51
F @ matchRoutes.js:231
c @ createTransitionManager.js:41
r @ createTransitionManager.js:222
listen @ createTransitionManager.js:246
componentWillMount @ Router.js:102
_o @ react-dom.production.min.js:137
$s @ react-dom.production.min.js:176
Oa @ react-dom.production.min.js:263
mc @ react-dom.production.min.js:246
bc @ react-dom.production.min.js:246
cc @ react-dom.production.min.js:239
rc @ react-dom.production.min.js:230
jc @ react-dom.production.min.js:281
(anonymous) @ react-dom.production.min.js:284
uc @ react-dom.production.min.js:240
Gc @ react-dom.production.min.js:284
t.render @ react-dom.production.min.js:290
T3 @ app.js:78
(anonymous) @ app.js:111
react-dom.production.min.js:28 Error: <svg> attribute width: A negative value is not valid. ("-24")
Z @ react-dom.production.min.js:28
ta @ react-dom.production.min.js:201
Mc @ react-dom.production.min.js:247
mc @ react-dom.production.min.js:246
bc @ react-dom.production.min.js:246
cc @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
lc @ react-dom.production.min.js:240
notify @ Subscription.js:16
notifyNestedSubs @ Subscription.js:88
i @ Subscription.js:93
g @ redux.js:296
(anonymous) @ session-middleware.js:53
(anonymous) @ middleware.js:22
(anonymous) @ index.js:28
(anonymous) @ store.js:21
dispatch @ redux.js:691
(anonymous) @ redux.js:206
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
Promise.then (async)
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ redux.js:202
(anonymous) @ store.js:19
dispatch @ redux.js:691
(anonymous) @ data-fetching.js:382
(anonymous) @ data-fetching.js:380
(anonymous) @ redux.js:204
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ redux.js:202
(anonymous) @ store.js:19
n.<computed> @ bindActionCreators.js:8
(anonymous) @ Dashboard.jsx:142
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
_R @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
ua @ react-dom.production.min.js:212
yc @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
zc @ react-dom.production.min.js:248
cc @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
lc @ react-dom.production.min.js:240
notify @ Subscription.js:16
notifyNestedSubs @ Subscription.js:88
i @ Subscription.js:93
g @ redux.js:296
(anonymous) @ session-middleware.js:53
(anonymous) @ middleware.js:22
(anonymous) @ index.js:28
(anonymous) @ store.js:21
dispatch @ redux.js:691
(anonymous) @ redux.js:206
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
Promise.then (async)
y @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ redux.js:202
(anonymous) @ store.js:19
n.<computed> @ bindActionCreators.js:8
(anonymous) @ Dashboard.jsx:170
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
_R @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ Dashboard.jsx:123
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
_R @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
ua @ react-dom.production.min.js:212
yc @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
zc @ react-dom.production.min.js:248
cc @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Yi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rc @ react-dom.production.min.js:230
enqueueSetState @ react-dom.production.min.js:132
v.setState @ react.production.min.js:12
(anonymous) @ Router.js:109
(anonymous) @ createTransitionManager.js:228
(anonymous) @ createTransitionManager.js:83
(anonymous) @ AsyncUtils.js:74
(anonymous) @ AsyncUtils.js:80
(anonymous) @ getComponents.js:6
(anonymous) @ getComponents.js:30
(anonymous) @ AsyncUtils.js:79
m @ AsyncUtils.js:78
x @ getComponents.js:29
u @ createTransitionManager.js:77
l @ AsyncUtils.js:47
(anonymous) @ TransitionUtils.js:88
(anonymous) @ TransitionUtils.js:110
(anonymous) @ routes.jsx:179
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
Rkt @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
Promise.then (async)
Rkt @ app-main.bundle.js?1cf9c4537e0911020cd2:2
a @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ app-main.bundle.js?1cf9c4537e0911020cd2:2
(anonymous) @ routes.jsx:176
o @ TransitionUtils.js:43
(anonymous) @ TransitionUtils.js:114
(anonymous) @ TransitionUtils.js:84
l @ AsyncUtils.js:34
g @ AsyncUtils.js:51
r @ TransitionUtils.js:83
runEnterHooks @ TransitionUtils.js:107
(anonymous) @ createTransitionManager.js:70
r @ TransitionUtils.js:74
runChangeHooks @ TransitionUtils.js:131
l @ createTransitionManager.js:67
(anonymous) @ createTransitionManager.js:45
l @ AsyncUtils.js:41
g @ AsyncUtils.js:51
F @ matchRoutes.js:231
c @ createTransitionManager.js:41
r @ createTransitionManager.js:222
listen @ createTransitionManager.js:246
componentWillMount @ Router.js:102
_o @ react-dom.production.min.js:137
$s @ react-dom.production.min.js:176
Oa @ react-dom.production.min.js:263
mc @ react-dom.production.min.js:246
bc @ react-dom.production.min.js:246
cc @ react-dom.production.min.js:239
rc @ react-dom.production.min.js:230
jc @ react-dom.production.min.js:281
(anonymous) @ react-dom.production.min.js:284
uc @ react-dom.production.min.js:240
Gc @ react-dom.production.min.js:284
t.render @ react-dom.production.min.js:290
T3 @ app.js:78
(anonymous) @ app.js:111
react-dom.production.min.js:28 Error: <svg> attribute height: A negative value is not valid. ("-24")
```
</details>

**To Reproduce**
Go to [dashboard 912](https://stats.metabase.com/dashboard/912) and pull the browser console

**Expected behavior**
Either fix the issue or don't show errors?

**Screenshots/videos**
NA

**Information about your Metabase Installation:**
- Your browser and the version: Brave latest
- Your operating system: Pop OS 22.04
- Metabase version: master

**Severity**
P3 for now

**Additional context**
NA

--- a/frontend/src/metabase/visualizations/visualizations/PieChart/PieChart.jsx
+++ b/frontend/src/metabase/visualizations/visualizations/PieChart/PieChart.jsx
@@ -408,7 +408,11 @@ export default class PieChart extends Component {
       slices.push(otherSlice);
     }
 
-    const side = Math.min(Math.min(width, height) - SIDE_PADDING, MAX_PIE_SIZE);
+    const side = Math.max(
+      Math.min(Math.min(width, height) - SIDE_PADDING, MAX_PIE_SIZE),
+      0,
+    );
+
     const outerRadius = side / 2;
     const labelFontSize = Math.max(
       MAX_LABEL_FONT_SIZE * (side / MAX_PIE_SIZE),

--- a/frontend/src/metabase/visualizations/visualizations/PieChart/PieChart.unit.spec.js
+++ b/frontend/src/metabase/visualizations/visualizations/PieChart/PieChart.unit.spec.js
@@ -84,7 +84,7 @@ describe("PieChart", () => {
     expect(screen.getByTestId("pie-chart")).toBeInTheDocument();
   });
 
-  it("should not have negative dimensions", () => {
+  it("should not have negative dimensions (metabase#28677)", () => {
     setup();
 
     const pieChart = screen.getByTestId("pie-chart");

--- a/frontend/src/metabase/visualizations/visualizations/PieChart/PieChart.unit.spec.js
+++ b/frontend/src/metabase/visualizations/visualizations/PieChart/PieChart.unit.spec.js
@@ -71,16 +71,28 @@ const setup = () => {
   renderWithProviders(<Container />);
 };
 
-describe("table settings", () => {
+describe("PieChart", () => {
   beforeAll(() => {
     //Append mocked style for .hide class
     const mockedStyle = document.createElement("style");
     mockedStyle.innerHTML = `.hide {display: none;}`;
     document.body.append(mockedStyle);
   });
+
   it("should render", () => {
     setup();
     expect(screen.getByTestId("pie-chart")).toBeInTheDocument();
+  });
+
+  it("should not have negative dimensions", () => {
+    setup();
+
+    const pieChart = screen.getByTestId("pie-chart");
+    const width = pieChart.getAttribute("width");
+    const height = pieChart.getAttribute("height");
+
+    expect(parseFloat(width)).toBeGreaterThanOrEqual(0);
+    expect(parseFloat(height)).toBeGreaterThanOrEqual(0);
   });
 
   it("should allow you to show and hide the grand total", async () => {

